### PR TITLE
added correct proptype and updated docpage

### PR DESCRIPTION
**Pull request by davecarter**
## PR Summary
- Updated doc page
- Added key to array items in docs

**Primary placeholder expects an Array of objects** - Don't pass plain objects as props in order to avoid invalid proptypes warning messages.

Please review: @kikoruiz @ruben-martin-lozano @aleanez

### CODE REVIEW

File: docs/index.jsx [hunk 1]
```diff
@@ -20,11 +20,11 @@ const images = [{
 
 ReactDom.render(
     <SuiCard className={'myCustom-class'}
-      primary={
-        <a href={images[0].link}><img src={images[0].src} /></a>
-      }
+      primary={[
+        <a key={1} href={images[0].link}><img src={images[0].src} /></a>
```
Comment (danderu) [inline docs/index.jsx]: I think that the sui-card should be the one that gives keys to its components, don't you?

Comment (danderu) [inline docs/index.jsx]: If you `.map` your array of primary and secondary components here: https://github.com/SUI-Components/sui-card/blob/master/src/sui-card/index.jsx#L34 you won't need to supply keys to your components in the moment of adding them to the array, the map function and its index will do the work for you.

**Comment by ruben-martin-lozano** [comment]
Did we only have to modify "docs" files? Not in src path?

File: docs/index.jsx (hunk 1 above)
Comment (kikoruiz) [inline docs/index.jsx]: Better if we get rid of these keys here.

**Comment by davecarter** [comment]
I agree with all of you.
As @ruben-martin-lozano mentioned before, is just an implementation detail in the `docs` page so Is not a big deal if a manual key is added as far as this pattern doesn't affect to the component's source folder.

**Comment by danderu** [comment]
@davecarter it is not a docs issue what we have found here. It's a `sui-card` implementation issue. The sui-card is not doing its job and giving a key to every component passed by parameter in `primary` and `secondary` arrays. It is delegating this responsibility to any of its implementers. In my opinion, it should not.

**Comment by davecarter** [comment]
@danderu ok
...
I'l take a look then..
:eyes:

**Comment by ruben-martin-lozano** [comment]
How is it going @davecarter ? :)

**Comment by davecarter** [comment]
The only way I know to get rid of these messages are mapping an array of objects passed into the primary & secondary sui-Card placeholders (and keying each mapped result). But if we do so we loose flexibility in the HTML structure.

In addition, If we decide this approach is not ok, we might consider refactoring the `sui-Topbar` component because the implementation is the same :(
https://github.com/SUI-Components/sui-topbar/blob/master/docs/index.jsx

If you think we need further discussion let's talk about it after daily
@ruben-martin-lozano @danderu @kikoruiz @carlosvillu

**Comment by danderu** [comment]
@davecarter yes, the implementation is the same but the use case is completely different. Look at how it's used: https://github.com/scm-spain/re-topbar/blob/master/src/re-topbar/index.jsx#L41

Let's discuss it in person today

**Comment by danderu** [comment]
Fixed! Now I like it :) :+1:

**Pull request merged**